src/index.js

```diff
@@ -0,0 +1,130 @@
+'use strict';
+
+let holup = Array();
+
+let state = {
+  temp: 0, // kelvin
+  city: 'Seattle',
+  condition: 'sleepless',
+  sky: 'the color of a television tuned to a dead channel',
+};
+
+const tempF = () => Math.round(1.8 * (state.temp - 273) + 32);
+const tempC = () => Math.round(state.temp - 273);
+
+const renderPage = () => {
+  const temperatureStyle = (F) => {
+    if (F > 80) {
+      return { bgColor: 'red', landscape: '🌵__🐍_🦂_🌵🌵__🐍_🏜_🦂' };
+    } else if (F >= 70) {
+      return { bgColor: 'orange', landscape: '🌸🌿🌼__🌷🌻🌿_☘️🌱_🌻🌷' };
+    } else if (F >= 60) {
+      return { bgColor: 'yellow', landscape: '🌾🌾_🍃_🪨__🛤_🌾🌾🌾_🍃' };
+    } else if (F >= 50) {
+      return { bgColor: 'green', landscape: '🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲' };
+    } else {
+      return { bgColor: 'teal', landscape: '🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲' };
+    }
+  };
```

Comment:
How might you refactor this so the logic is more data driven? Here you have several if/else if statements. What if you had an object with the keys as the temperature and the values as the `landscape` and `bgColor` corresponding display? Then you could iterate over the object to set state!

It could look something like this:
```js
 const bgColors = [
   [49, 'blue'],
   [59, 'green'],
   [69, 'gold'],
   [79, 'orange'],
   [null, 'red'],
 ];
 
 const getColorForTemp = (temp) => {
   for (const [boundaryTemp, color] of tempColors) {
     if (boundaryTemp === null || temp <= boundaryTemp) {
       return color;
     }
   }
 };
  ```